plugins/dbviewer/frontend/public/localization/dbviewer.properties

```diff
@@ -19,4 +19,5 @@ dbviewer.descending = Descending
 dbviewer.countly-database = Countly Database
 dbviewer.countly-database-description = Common aggregated anonymous statistics on users and their segments.
 dbviewer.countly-drill-database = Countly Drill Database
-dbviewer.countly-drill-database-description = More granular data and data relations for users and their actions.
\ No newline at end of file
+dbviewer.countly-drill-database-description = More granular data and data relations for users and their actions.
+dbviewer.all-apps = All apps
```

Comment:
I think it would be enough to reuse https://github.com/Countly/countly-server/blob/master/frontend/express/public/localization/dashboard/dashboard.properties#L35
no need to create another translation string